pages/home/build-tools/create-a-toolkit.mdx

```diff
@@ -196,32 +203,64 @@ arcade show --local
 
 You should see your toolkit listed in the output.
 
-### Test with a local worker
+### Test your tools locally
 
-Start a local worker to test your toolkit with the Arcade API:
+Serve your toolkit locally with the Arcade CLI:
 
 ```bash
 arcade serve
 ```
 
-Visit http://localhost:8002/docs to see your tools in the Swagger UI.
+Visit http://localhost:8002/worker/health to see that your worker is running.
+
+### Connect your local toolkit to Arcade Gateway
+
+In another terminal, use a service like [ngrok](https://ngrok.com/docs/), [tailscale](https://tailscale.com/kb/1223/funnel), or [cloudflare](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/do-more-with-tunnels/local-management/create-local-tunnel/) to give your local worker a public URL.
+
+```bash
+ngrok http 8002
+```
+
+```bash
+tailscale funnel 8002
+```
+
+```bash
+cloudflared tunnel --url http://localhost:8002
+```
+
+Then in your Arcade dashboard:
+
+1. Navigate to the [Workers](https://api.arcade.dev/dashboard/workers) page.
+1. Click **Add Worker**.
+1. Fill in the form with the following values:
+    - **ID**: `my-arithmetic-worker`
+    - **Worker Type**: `Arcade`
+    - **URL**: your public URL from ngrok, tailscale, or cloudflare
+    - **Secret**: `dev`
+    - **Timeout** and **Retry**: can be left at default values
+1. Click **Create**.
+
+Now, in the [Playground](https://api.arcade.dev/playground/execute), you can see your tools in action.
 
 ### Deploy your toolkit to Arcade
 
-Create a `worker.toml` file in your toolkit's root directory:
+Find the `worker.toml` file in your toolkit's root directory (where you ran `arcade new`):
```

Comment:
We should point them somewhere else if they're using ngrok or tailscale. Should be able to do `arcade serve` instead of using deploy

Reply:
Added a clarifying sentence: https://github.com/ArcadeAI/docs/pull/260/commits/e22e8b6aa889ba7e63e631e1f177609071eabec9

The `arcade serve` is just before this.

Reply:
Is that better?